includes/remote-config/class-wc-stripe-remote-config.php

```
@@ -0,0 +1,208 @@
+	 * Resolve a flag's value: remote wins if present, otherwise the local fallback.
+	 *
+	 * @param string      $flag        Flag name as declared in WC_Stripe_Remote_Config_Flags::FLAGS.
+	 * @param mixed       $local_value Caller-computed local value (the existing logic's result).
+	 * @param string|null $mode        'live'/'test'/null (auto-derive).
+	 *
+	 * @return mixed
+	 */
+	public function resolve( string $flag, $local_value, ?string $mode = null ) {
+		$remote = $this->get_flag( $flag, $mode );
+		return null === $remote ? $local_value : $remote;
+	}
```

PHPMD (2.15.0)

[error] The parameter $local_value is not named in camelCase. (undefined)

(CamelCaseParameterName)